ports/directx-dxc/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "directx-dxc",
+  "version-date": "2021-12-08",
+  "description": "DirectX Shader Compiler (LLVM/Clang)",
+  "homepage": "https://github.com/microsoft/DirectXShaderCompiler",
+  "documentation": "https://github.com/microsoft/DirectXShaderCompiler/wiki",
+  "license": null,
+  "supports": "windows & x64 & !uwp"
```

```suggestion
  "supports": "windows & x64 & !uwp & !static"
```
This port is ONLY_DYNAMIC_LIBRARY, so it shouldn't support static.

---

Actually I believe it has to be ``& !staticcrt``

---

Fixed

---

Yes, `!staticcrt` matches the triplets you mentioned better.